hc/settings.py

```diff
@@ -120,8 +93,8 @@
 
 USE_TZ = True
 
-SITE_ROOT = "http://localhost:8000"
-SITE_NAME = "healthchecks.io"
+SITE_ROOT = "https://hc-bs-staging.herokuapp.com/"
```

Comment:
@collinmutembei i can only see this on hc/setting.py
`WSGI_APPLICATION = 'hc.wsgi.application'
TEST_RUNNER = 'hc.api.tests.CustomRunner'

LANGUAGE_CODE = 'en-us'

TIME_ZONE = 'UTC'

USE_I18N = True

USE_L10N = True

USE_TZ = True

SITE_ROOT = "https://hc-bs-staging.herokuapp.com/"
SITE_NAME = "My Monitoring Project"
PING_ENDPOINT = SITE_ROOT + "/ping/"
PING_EMAIL_DOMAIN = HOST
STATIC_URL = '/static/'
STATICFILES_DIRS = [os.path.join(BASE_DIR, "static")]
STATIC_ROOT = os.path.join(BASE_DIR, 'static-collected')
STATICFILES_FINDERS = (
    'django.contrib.staticfiles.finders.FileSystemFinder',
    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
    'compressor.finders.CompressorFinder',
)
COMPRESS_OFFLINE = True
`
am i looking at the wrong place?